<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajedrez</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#MainWindow{

	
	background-color: rgb(98, 160, 67);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>580</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Colocar Fichas</string>
       </property>
      </widget>
      <widget class="QPushButton" name="resignW">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>290</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	text-align:center;
    font-family: 'Helvetica Neue', Helvetica, sans-serif;
	color: #FFF;
	background: #7F8C8D;
	border-radius: 10px;

}

QPushButton:hover{
  background: #ff5242;
  background: #E74C3C;
}
</string>
       </property>
       <property name="text">
        <string>Rendirse</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/new/img/img/resigning.png</normaloff>:/new/img/img/resigning.png</iconset>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumberW">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans Ultra Bold Condensed</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QWidget" name="blackCaptured" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>181</width>
         <height>181</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#blackCaptured{
	border: 3px solid rgb(0, 85, 0);
}</string>
       </property>
      </widget>
      <widget class="QLabel" name="wT">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans Ultra Bold Condensed</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(212, 203, 100);</string>
       </property>
       <property name="text">
        <string>Turno Blancas</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans Ultra Bold Condensed</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Jugador 1</string>
       </property>
      </widget>
      <widget class="QPushButton" name="menu">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>460</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans Ultra Bold Condensed</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 77, 46, 255));
border-radius: 10px;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Ir a men√∫</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="Board" name="board" native="true">
      <property name="minimumSize">
       <size>
        <width>610</width>
        <height>610</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>610</width>
        <height>610</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <widget class="QPushButton" name="resignB">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>290</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	text-align:center;
    font-family: 'Helvetica Neue', Helvetica, sans-serif;
	color: #FFF;
	background: #7F8C8D;
	border-radius: 10px;

}

QPushButton:hover{
  background: #ff5242;
  background: #E74C3C;
}
</string>
       </property>
       <property name="text">
        <string>Rendirse</string>
       </property>
       <property name="icon">
        <iconset resource="img.qrc">
         <normaloff>:/new/img/img/resigning.png</normaloff>:/new/img/img/resigning.png</iconset>
       </property>
      </widget>
      <widget class="QLCDNumber" name="lcdNumberB">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans Ultra Bold Condensed</family>
        </font>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>430</y>
         <width>181</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans MT</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
      </widget>
      <widget class="QWidget" name="whiteCaptured" native="true">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>181</width>
         <height>181</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#whiteCaptured{
border: 3px solid rgb(0, 85, 0);
}</string>
       </property>
      </widget>
      <widget class="QLabel" name="bT">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans Ultra Bold Condensed</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Turno Negras</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>10</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans Ultra Bold Condensed</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Jugador 2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>390</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gill Sans Ultra Bold Condensed</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Registro:</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Board</class>
   <extends>QWidget</extends>
   <header>board.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
